src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts

```diff
@@ -55,11 +55,31 @@ export class ExuiCcdConnectorComponent implements AfterContentInit, OnDestroy {
     this.dispatcherContainer = {};
     this.eventsBindings.forEach(event => {
       this.dispatcherContainer[event.type] = (obj) => {
-        this.store.dispatch(new this.fromFeatureStore[event.action](obj));
+        this.store.dispatch(new this.fromFeatureStore[event.action](this.deepClone(obj)));
       };
     });
   }
 
+  deepClone(obj) {
+    return JSON.parse(JSON.stringify(this.simplifyFormGroup(obj)));
+  }
+
+  simplifyFormGroup(obj) {
```

Comment:
I understand this a temporary code till CCD team comes up with a fix. Can you add a comment to highlight that this is temporary fix? Thanks.

Reply:
It is unlikely that this will be fixed by CCD. It is not their concern.

Reply:
fyi - We had a meeting with CCD yesterday and discussed this issue with them. They are looking into this.